lib/defaults.js

```diff
@@ -1,21 +1,14 @@
 'use strict';
-
-const controllers = require('hof-controllers');
-const path = require('path');
-
-const testPath = path.resolve(__dirname, '../test/');
-const realPath = path.dirname(module.parent.parent.filename);
 /* eslint no-process-env: "off" */
-const caller = process.env.NODE_ENV === 'test' ? testPath : realPath;
 
 const defaults = {
+  root: process.cwd(),
   translations: 'translations',
-  caller: caller,
   start: true,
   getCookies: true,
   getTerms: true,
   viewEngine: 'html',
-  baseController: controllers.base,
+  baseController: require('hof-controllers').base,
```

Comment:
I largely agree, except for short files, and in particular where no function calls are made. As soon as you're inside a function scope then you should be require-ing anything because blocking, but for something like this where everything is executed immediately my official position is ¯\_(ツ)_/¯